db/migrate/20230321055733_add_index_to_ahoy_events_name_and_time.rb

```diff
@@ -0,0 +1,6 @@
+class AddIndexToAhoyEventsNameAndTime < ActiveRecord::Migration[6.1]
+  def change
+    add_index :ahoy_events, :name
+    add_index :ahoy_events, :time
```

Comment:
this looks like a long running migration, we should get @Pralish 's `strong_migrations` changes into master so we could calculate the risk of this migration